src/containers/Guide/GuideInitialFunds/index.tsx

```diff
@@ -21,7 +21,7 @@ const GuideInitialFunds: FC = () => {
         first. Given that the usefulness of currency is the transfer of value from one individual to another, the
         initial distribution of funds must be to several individuals. This would not only help alleviate the
         concentration of wealth that is apparent in the majority of existing economies, both fiat and digital but also
-        aid early adoption by placing the currency in the hands of many rather than a select few. With the foundational
+        aid early adoption by placing the currency in the hands of many rather than a selected few. With the foundational
```

Comment:
This can remain select few only. [Reason](https://www.quora.com/Which-is-correct-this-offer-is-only-for-a-selected-few-or-this-offer-is-only-for-a-select-few#:~:text=The%20correct%20one%20is%20%E2%80%9CThis,(for%20this%20particular%20offer).&text=It%20is%20still%20a%20verb%2C%20but%20in%20the%20past%20participle%20form.)